workers/data_refinery_workers/processors/management/commands/create_compendia.py

```diff
@@ -83,16 +89,21 @@ def handle(self, *args, **options):
         if options["quant_sf_only"] is True:
             quant_sf_only = True
 
+        # default algorithm to arpack until we decide that ranomized is preferred
+        svd_algorithm = 'ARPACK'
```

Comment:
This should default to None if we're making a quantpendia. 
```suggestion
        svd_algorithm = 'NONE' if quant_sf_only else 'ARPACK' 
```